From: SJ Park <sj@kernel.org>
Date: Sun, 2 Aug 2026 11:42:22 -0700
Subject: [PATCH] ==== commit_inputs infinite hang fix ====

mm/damon/{reclaim,lru_sort}: fix commit_inputs infinite hang

Writing 'Y' to commit_inputs parameters of DAMON_RECLAIM and
DAMON_LRU_SORT before the modules were ever turned on causes infinite
hang.  Fix those.

The issue was discovered [1] by Sashiko.

[1] https://lore.kernel.org/20260802173021.762-1-sj@kernel.org

Changes from v1
- v1: https://lore.kernel.org/20260803134646.16640-1-sj@kernel.org
- None.
Changes from RFC
- RFC: https://lore.kernel.org/20260802185430.9940-1-sj@kernel.org
- Drop RFC tag.

Signed-off-by: SJ Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_26cbisfo | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_26cbisfo

diff --git a/hkml_cv_bogus/hkml_cv_bogus_26cbisfo b/hkml_cv_bogus/hkml_cv_bogus_26cbisfo
new file mode 100644
index 0000000000000..e69de29bb2d1d
-- 
2.47.3

